connectors/src/connectors/confluence/index.ts

```diff
@@ -73,8 +74,7 @@ export async function createConfluenceConnector(
       return connector;
     });
 
-    // TODO(2024-01-10 flav) Uncomment in next PR.
-    // await launchConfluenceFullSyncWorkflow(connector.id, null);
+    await launchConfluenceFullSyncWorkflow(connector.id, null);
```

Comment:
Need an Ok / Err to check if the workflow has been launched or not.

Reply:
Fixed.